ext/standard/tests/array/array_key_first.phpt

```diff
@@ -0,0 +1,214 @@
+--TEST--
+Test array_key_first() function
+--FILE--
+<?php
+
+array_key_first($GLOBALS);
+
+/* Various combinations of arrays to be used for the test */
+$mixed_array = array(
+  array(),
```

Comment:
You should add test cases with:

- A single element here: ``array( "foo" );``
- An array where there is one element, and it doesn't start with index 0: ``array( 1 => "42" ),``

Reply:
Good additional cases, I'll add them to the tests for all four functions.

Thanks for the input!

Reply:
I've added the cases to the accepted two functions